app/controllers/teams_controller.rb

```diff
@@ -47,6 +48,7 @@ def create
 
   def update
     @conferences = conference_list
+    @conference = Conference.new
```

Comment:
I'd prefer to now assign another view variable, see my comment in the form partial 👇